CMakeLists.txt

```diff
@@ -4,6 +4,8 @@
 cmake_minimum_required (VERSION 2.8)
 cmake_policy (VERSION 2.8)
 
+option (UBUNTU_BASED_DISTRO "Whether to build in systemwide proxy support, depends on ubuntu-system-service" ON)
```

Comment:
Could we change this name to something like `USE_UBUNTU_SERVICES`? I'm also fine with this though.

Reply:
If we're going to keep this for now, make it `USE_UBUNTU_SERVICES` and have it off by default.